R/extract_hashtags.R

```diff
@@ -12,6 +12,14 @@
 #' extract_hashtags(tweets)
 #'
 extract_hashtags <- function(tweets) {
-
+  if(!is.character(tweets)) {
+    stop("input should be a character vector")
+  }
+  if(length(tweets)==0) {
+    stop("input vector cannot be null")
```

Comment:
I'm not sure what you mean by cannot be null

Reply:
oh my! That's non sense. Will remove it